read-fonts/src/tables/variations.rs

```diff
@@ -275,6 +300,7 @@ impl<'a> PackedPointNumbers<'a> {
     }
 
     /// the number of bytes to encode the packed point numbers
+    #[inline(never)]
```

Comment:
?

Reply:
Curiously, I added these to generate finer grained profiles, but ended up with consistent and measurable performance gains due to the lack of inlining. I'm chalking this up to sometimes the optimizer blows the inlining budget before it gets deep enough into the call graph to reach functions where const prop actually makes a huge difference.